packages/mobile/src/pincode/PincodeEnter.tsx

```diff
@@ -57,13 +51,11 @@ class PincodeEnter extends React.Component<Props, State> {
   }
 
   onWrongPin = () => {
```

Comment:
this looks like it will be easy to do with these changes, but just a note that i'd like to limit the number of incorrect entries before someone gets locked out for a while

Reply:
👍